app/assets/javascripts/app/lib/app_post/searchable_select.coffee

```diff
@@ -21,6 +21,7 @@ class App.SearchableSelect extends Spine.Controller
     '.js-input':                  'input'
     '.js-shadow':                 'shadowInput'
     '.js-optionsList':            'optionsList'
+    '.items':                     'itemList'
```

Comment:
This is only used in a descendant of the class so there is no need to have it in the base class.